src/ROUTES.js

```diff
@@ -43,13 +43,14 @@ export default {
      * @returns {Object}
      */
     parseReportRouteParams: (route) => {
-        if (!route.startsWith(wrapWithForwardSlash(REPORT))) {
+        if (!route.startsWith(addTrailingForwardSlash(REPORT))) {
             return {};
         }
 
         const pathSegments = route.split('/');
         return {
             reportID: lodashGet(pathSegments, 1),
+            isParticipantsRoute: Boolean(lodashGet(pathSegments, 2)),
```

Comment:
Coming from [issue - app crash when url is invalid](https://github.com/Expensify/App/issues/22507)

If the url has multiple double slash eg: `https://.....https//`, `pathSegments[2]` will be false. 
We should have simply checked the length of the array 

More details here - https://github.com/Expensify/App/issues/22507#issuecomment-1650301007